### PR TITLE
Update calling-plans-for-office-365.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2206
Fixed link

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/calling-plans-for-office-365.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Here are more articles that explain how to set up your Calling Plans:
     
 - [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md)
     
-- [Plan your Phone System in Office 365 (Cloud PBX) solution](https://go.microsoft.com/fwlink/p/?LinkId=717926)
+- [Plan your Phone System in Office 365 (Cloud PBX) solution](https://docs.microsoft.com/MicrosoftTeams/here-s-what-you-get-with-phone-system)
```
Comment (nenonix) [inline Teams/calling-plans-for-office-365.md]: The link text should also be changed, this new link does not go where the text describes

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@CarolynRowe Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**